# BI_flora_thesis_appendices
## Appendices and supplementary material for thesis

This repository contains additional apendices for my thesis that are more accessible in file format.

## Appendix 1 - Supporting information for Chapter 2

**Supporting Table S2.3 Full reference list for generation of phylogeny.** The taxa included are those 659 species that could not be attached to the phylogeny directly. 161 species were attached as unambiguous synonyms (via WCVP information). The remaining 497 species were either attached directly to a node if they were the only other species of the same genus, the only member of a genus, or if a clear location for the species could reasonably be inferred from previous phylogenetic studies. 151 species could not be assigned a clear location in the phylogeny and were not attached. The action taken to assign each species a position in the phylogeny is explained in column ‘Attachment rationale’. Phylogenetic information sources are referenced in column ‘Citation for phylogeny’. ITS_corrected refers to the unpublished phylogeny shared with me by Max R Brown and Alex Twyford. 

**Supporting Table S2.4 CSR scores calculated using the method by Pierce *et al.*, 2017.** Columns describing taxon_name and kew_id correspond to BIFloraExplorer accessions, C_num, S_num and R_num describe the competitor, stress-tolerator and ruderal scores (out of 100), and predicted_CSR shows the final category the species is classed into, according to Grime's (1974) CSR strategies and as imputed with the method by Pierce *et al.*, 2017. Only species with full data coverage are included (1,441 species).

**Supporting Method S2.1 Phylogenetic tree of BI species as TREE file.** 2,501 tips.

## Appendix 2 - Supporting information for Chapter 3

### Frescalo output tables. (folder Frescalo output tables)
Column headers are explained in detail under https://rdrr.io/github/BiologicalRecordsCentre/sparta/man/frescalo.html.
Species are identified by their kew_id as per https://github.com/RBGKew/BIFloraExplorer and https://powo.science.kew.org/.

**Supporting Table S3.1 Dataframe containing the results of linear regressions on time factors across three date classes (1987-1999, 2000-2009, 2010-2019).**


**Supporting Table S3.2a Dataframe listing time factors calculated for each species (1987-1999).**

**Supporting Table S3.2b Location report for each hectad (1987-1999).**

**Supporting Table S3.2c Listing of rescaled species frequencies per hectad (1987-1999).**


**Supporting Table S3.3a Dataframe listing time factors calculated for each species (2000-2009).**

**Supporting Table S3.3b Location report for each hectad (2000-2009).**

**Supporting Table S3.3c Listing of rescaled species frequencies per hectad (2000-2009).**


**Supporting Table S3.4a Dataframe listing time factors calculated for each species (2010-2019).**

**Supporting Table S3.4b Location report for each hectad (2010-2019).**

**Supporting Table S3.4c Listing of rescaled species frequencies per hectad (2010-2019).**


### Frescalo output maps.
**Supporting Figure S3.1 Maps and regression graphs of Frescalo outputs across three date classes (1987-1999, 2000-2009, 2010-2019).** Page one shows summary statistics for the overall run (a, the raw species numbers from distribution records; b, the rescaled number of species following Frescalo correction; c and d show the the local scaling factor alpha with derived and specified breaks used for plotting). Each following page shows an overview of the results obtained for each species with sufficient data. This overview consists of graphs of the actual distribution of species, i.e. all hectads with records during any of the three date classes (a), estimated neighbourhood frequencies (b) and final adjusted species frequencies (c) as well as a plot illustrating the regression of time factors across the three date classes (d). 

## Appendix 3 - Supporting information for Chapter 4

**Supporting Table S4.1 Full genomic information compiled for the BI flora including assumptions.** The information was compiled as outlined in Chapter 4. kew_id and taxon_name correspond to BIFloraExplorer. GS columns and assumed_ploidy and chromosome_number give the the values of genomic characters used in the thesis. Columns assumptions_made and assumption_notes outline if and what assumptions were made on what basis to obtain full genomic infirmation. The last column (source gives information regarding the publication or repository that held the genomic data used. Cytotypes are included if there was sufficient support for their existence.

**Supporting Table S4.1 Summary of available genomic information used for the creation of Supporting Table S4.1.** This table encompasses the data used to derive the full genomic information used in the thesis. No assumptions were made here. Columns kew_id, taxon_name, sporophytic_chromosome_number and infraspecific_variation_chrom_number correspond to BIFloraExplorer. Column matched_by details if the genomic information is associated directly with the taxon name accepted by Stace, 2019 (matched_by = Stace), by the WCVP accepted name, by a synonym or subspecies. If data was associated with anything but the Stace species name, column matched_name gives the taxon name the genomic data was published for. Columns from GS_2C_pg to chromosome_number give published genomic information. Source notes the publication or repository the information was derived from. The final column (chrom_num_compatible) assesses if the published chromosome count is in agreement with data verified by Richard Gornall. 

## Appendix 4 - Supporting information for Chapter 5
